tesseract-core/src/sql.rs

```diff
@@ -33,9 +33,9 @@ pub(crate) fn standard_sql(
     // hack for now... remove later
     fn agg_sql_string(m: &MeasureSql) -> String {
         match &m.aggregator {
-            Aggregator::Sum => format!("sum"),
-            Aggregator::Count => format!("count"),
-            Aggregator::Average => format!("avg"),
+            Aggregator::Sum => format!("sum({})", &m.column),
```

Comment:
Thanks for the fixes here; I'm not working much with general sql yet, so this is very helpful